apps/vscode/command_client/command_client.py

```diff
@@ -26,13 +26,12 @@
 
 mod = Module()
 
-global_ctx = Context()
 ctx = Context()
 mac_ctx = Context()
 linux_ctx = Context()
 
 ctx.matches = r"""
-app: vscode
+tag: command_client
```

Comment:
This is wrong! It needs to be `tag: user.command_client`. Ugh, talon should warn about this.

Reply:
fixed it.